src/routes/LogInForm.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import users from "../localDB/users.json";
+import { useNavigate } from "react-router-dom";
+import "./login.css";
+
+export default function LogInForm() {
+  const [usersData, setUsersData] = React.useState(users);
```

Comment:
I would create a users type inside the models directory and here we are going to have useState<user[]>(users)